.jshintrc

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
We should be using eslint (extending frost-standard) instead of jshint - let's just raise an issue and move on for now.